_dockerignore

```diff
@@ -0,0 +1,2 @@
+.git
```

Comment:
I'm not sure what a `_dockerignore` file is. Did you mean a `.dockerignore` file?